sdk/head_tracker.h

```diff
@@ -83,6 +91,10 @@ class HeadTracker {
   // Callback functions registered to the input SingleTypeEventProducer.
   std::function<void(AccelerometerData)> on_accel_callback_;
   std::function<void(GyroscopeData)> on_gyro_callback_;
+
+  // Quaternion to recenter the head tracker.
+  // It is initialized as an identity rotation.
+  Rotation recenter_rotation_ = Rotation();
```

Comment:
Is it required to explicitly call the constructor and copy/assign the constructed object?

Reply:
In other words, why not just `Rotation recenter_rotation_ `?